lua/entities/gmod_wire_cameracontroller.lua

```diff
@@ -431,6 +431,9 @@ end
 
 util.AddNetworkString( "wire_camera_controller_toggle" )
 function ENT:SyncSettings( ply, active )
+	if CurTime() < self.NextSync then return end
```

Comment:
This won't work properly because it'll not sync the most recent change. If you change for example two inputs nearly at the same time, only the 1st one will be synced.
You solved this by making SyncSettings get called every time SyncPositions is called, which isn't a good fix since that'll make this function be called many many more times than before this patch.

Instead, either create a second `NeedsSync` variable specifically for SyncSettings.